components/org.wso2.carbon.consent.mgt.core/src/main/java/org/wso2/carbon/consent/mgt/core/ConsentManager.java

```diff
@@ -405,24 +405,105 @@ public Receipt getReceipt(String receiptId) throws ConsentManagementException {
         return receipt;
     }
 
+    /**
+     * This API is used to select the PIIController from List of registered PIIController. By default set the height
+     * priority one.
+     * @param piiControllers list of PIIControllers.
+     * @return selected PIIController.
+     */
+    protected PIIController getPIIController(List<PIIController> piiControllers) {
```

Comment:
Since ConsentManager is not an extension point shall we make it private ?

Reply:
This can be an extension point in future. Hence better to keep this.